src/snowcli/config.py

```diff
@@ -54,17 +54,20 @@ def section_exists(self, *path) -> bool:
         except NonExistentKey:
             return False
 
-    def _get_from_env(self, *path, key: str):
-        env_variable_name = (
-            "SNOWFLAKE_" + "_".join(p.upper() for p in path) + f"_{key.upper()}"
+    @staticmethod
+    def _get_from_env(*path, connection_name: str) -> dict:
+        env_variable_prefix = (
+            "SNOWFLAKE_"
+            + "_".join(p.upper() for p in path)
+            + f"_{connection_name.upper()}"
         )
-        return os.environ.get(env_variable_name)
+        return {
+            k.replace(f"{env_variable_prefix}_", "").lower(): os.environ[k]
+            for k in os.environ.keys()
+            if k.startswith(env_variable_prefix)
+        }
 
     def get(self, *path, key: str, default: Optional[Any] = None) -> Any:
```

Comment:
```suggestion
    def get_from_config_file(self, *path, key: str, default: Optional[Any] = None) -> Any:
```

Reply:
I think we should also consider similar change for `get_section` and `section_exists`.